### ISSUE TITLE
ModularCurve

### ISSUE THREAD

**Issue by jvoight** [assaferan/ModFrmGL2#4]
This function (line 1174 in ModSym/twists.m) doesn't work for me:
```
ModularCurve(Gamma0(11));
```
gives
```
ModularCurve(
    G: Gamma_0(11)
)
CremonaMethod(
    H: MatrixGroup(2, IntegerRing(11)),
    ps_prec: 22,
    n_rel: 6
)
get_eigenvector_in_subspace(
    H: MatrixGroup(2, IntegerRing(11)),
    T: [1/5 0 0 -1/5 0 1/5 -1/5 -2/5 3/5 0 0 -1/5 -1/5 0 -1/5 2/5 0...,
    S: [0 0 0 0 0 0 1 -1 -1 0 0 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0...,
    Conj: [0 0 0 0 -1 0 0 -1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0...,
    M: Full modular symbols space of weight 2 and dimension 111 ove...
)
In file "/Users/jvoight/Dropbox/github/ModFrmGL2/ModFrmGL2/ModSym/twists.m", 
line 583, column 25:
>>   V_M := VectorSpace(S_H[1]);
                           ^
Runtime error in '[]': Illegal null sequence

```

I know it was designed to handle more general curves, but it would be nice if it called the easier method for Gamma_0(N) and Gamma_1(N)?

**Comment by assaferan** [comment]
It took a while, but I got to it. 
It can now also work for more non-trivial curves.

**Comment by assaferan** [comment]
This should have been resolved two years ago,, but also now in PR #24.
